cmd/ocm-support/patch/organization/cmd.go

```diff
@@ -0,0 +1,70 @@
+package organization
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/openshift-online/ocm-cli/pkg/ocm"
+	"github.com/spf13/cobra"
+
+	"github.com/openshift-online/ocm-support-cli/cmd/ocm-support/utils"
+	"github.com/openshift-online/ocm-support-cli/pkg/organization"
+	"github.com/openshift-online/ocm-support-cli/pkg/request"
+)
+
+var args struct {
+	dryRun     bool
+	maxRecords int
+}
+
+// CmdPatchOrganization represents the organization patch command
+var CmdPatchOrganization = &cobra.Command{
+	Use:     "organization [id]",
+	Aliases: utils.Aliases["organization"],
+	Short:   "Patches an organization for the given ID",
+	Long:    "Patches an organization for the given ID",
+	RunE:    run,
+	Args:    cobra.MaximumNArgs(1),
+}
+
+func init() {
+	flags := CmdPatchOrganization.Flags()
+	flags.BoolVar(
+		&args.dryRun,
+		"dryRun",
+		true,
+		"If false, it will execute the patch command call in instead of a dry run.",
+	)
+}
+
+func run(cmd *cobra.Command, argv []string) error {
+	connection, err := ocm.NewConnection().Build()
+	if err != nil {
+		return fmt.Errorf("failed to create OCM connection: %v", err)
+	}
+	if len(argv) != 1 {
+		return fmt.Errorf("expected exactly one argument")
+	}
```

Comment:
Using an empty argument here produces the following behaviour:
```

echo '{"Status":"Active"}'| ocm support patch org ""          
DRYRUN: Would have called PATCH on 
organization  would have been patched
```
Can we improve that as well? Similar to the filter argument treatment.